src/index.ts

```diff
@@ -487,7 +487,15 @@ class NotificationAPI {
 
       const date = document.createElement('p');
       date.classList.add('notificationapi-notification-date');
-      date.innerHTML = timeAgo.format(new Date(n.date), 'round-minute');
+      if (
+        new Date(n.date).getTime() - Date.now() > 0 &&
+        new Date(n.date).getTime() - Date.now() < 60 * 1000
+      ) {
+        date.innerHTML = timeAgo.format(Date.now(), 'round-minute');
+      } else {
+        date.innerHTML = timeAgo.format(new Date(n.date), 'round-minute');
+      }
```

Comment:
Just realized a simpler solution with less computation:

if(string === 'in a moment')
   date.innerHTML = 'just now'

What do you think?

Reply:
It's possible to, because in case, let say +70s, it will return "in 1 minute"

Reply:
@sahandseifi applied